### PR TITLE
Add linter configurations

### PR DESCRIPTION
**Pull request by sladyn98**
This ESLint config meets the following requirements:

a. Includes the recommended ESLint rules
b. Includes jest’s, es6’s and browser’s global variables
c. Disallows the use of eval
d. Warns the user if it uses console.log
e. Includes the import plugin and reports any imports that come after non-import statements
f. Indicates the parser you’re using ES2018’s syntax

### CODE REVIEW

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "es6": true
+    },
+    "plugins": ["import"],
+    "extends": "eslint:recommended",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "rules": {
+      "no-console": "warn",
```
Comment (kwhetstone) [inline frontend/.eslintrc.json:14]: 👍

File: frontend/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "no-eval": "error",
+      "import/first": "error"
+    }
+  }
```
Comment (kwhetstone) [inline frontend/.eslintrc.json:18]: Like LinuxSuRen  mentioned in another review, space at the end of the file.

Comment (sladyn98) [reply frontend/.eslintrc.json:18]: Yep thanks keep forgetting it :P

Comment (halkeye) [reply frontend/.eslintrc.json]: I would recommend switching to .eslintrc.js which then can be handled by eslint itself

**Pull request merged**